tools/plugins/com.liferay.ide.project.ui/resources/codeassitant.json

```diff
@@ -0,0 +1,322 @@
+[
```

Comment:
How is this file generated?  Is it edited by hand or is it generated some other way?